ports/arrow/vcpkg.json

```
@@ -0,0 +1,87 @@
+{
+  "name": "arrow",
+  "version": "4.0.0",
+  "description": "Cross-language development platform for in-memory analytics",
+  "homepage": "https://arrow.apache.org",
+  "dependencies": [
+    "boost-filesystem",
+    "boost-multiprecision",
+    "boost-system",
+    "brotli",
+    "bzip2",
+    "gflags",
+    "glog",
+    "lz4",
+    "openssl",
+    "re2",
+    "snappy",
+    "thrift",
+    "utf8proc",
+    "zlib",
+    "zstd"
+  ],
+  "default-features": [
+    "csv",
+    "dataset",
+    "filesystem",
+    "json",
+    "parquet"
+  ],
+  "features": {
+    "csv": {
+      "description": "CSV support"
+    },
+    "dataset": {
+      "description": "Dataset support"
+    },
+    "filesystem": {
+      "description": "Filesystem support"
+    },
+    "flight": {
+      "description": "Arrow Flight RPC support",
+      "dependencies": [
+        "abseil",
+        "c-ares",
+        "grpc",
+        "protobuf"
+      ]
+    },
+    "jemalloc": {
+      "description": "jemalloc allocator"
+    },
+    "json": {
+      "description": "JSON support",
+      "dependencies": [
+        "rapidjson"
+      ]
+    },
+    "mimalloc": {
+      "description": "mimalloc allocator"
+    },
+    "orc": {
+      "description": "ORC support",
+      "dependencies": [
+        "orc"
+      ]
+    },
+    "parquet": {
+      "description": "Parquet support"
+    },
+    "s3": {
+      "description": "S3 support",
+      "dependencies": [
+        {
+          "name": "aws-sdk-cpp",
```

```suggestion
          "name": "aws-sdk-cpp",
          "default-features": false,
```
Disabling the defaults is needed to avoid an unexpected dependency on dynamo.